### PR TITLE
docs: add release history section

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: releases/history.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Release History
+
+## Version 2.0
+* ### Latest Release: [v2.0.0](https://github.com/openebs/mayastor/releases/tag/v2.0.0)
+  * released: 
+* ### Patch Releases:
```
Comment (avishnu) [inline releases/history.md:6]: Don't we need to fill this space with the patch releases upto 1.0.5 ?

Comment (anupriya0703) [reply releases/history.md:6]: This PR is for 2.0.0 Hence, as and when we''ll have patch release for 2.0 (2.0.x) I will be adding it here. Is that alright?

Comment (avishnu) [reply releases/history.md:6]: I see. Ok

**Comment by anupriya0703** [comment]
The above changes are a part of https://github.com/openebs/mayastor-docs/pull/89.

**Pull request closed without merging**